apps/ledger-live-mobile/package.json

```diff
@@ -19,8 +19,8 @@
     "gen-metafile": "zx ./scripts/gen-metafile.mjs",
     "e2e:build": "pnpm detox build",
     "e2e:ci": "zx ./scripts/e2e-ci.mjs",
-    "e2e:test": "export MOCK=1 && pnpm detox test",
-    "e2e:test:speculos": "export MOCK=0 && pnpm detox test --testMatch \"$(pwd)/e2e/specs/speculos/{,**/,**/**/,**/**/**/}*.spec.ts\" --testTimeout=300000",
+    "mock:test": "export MOCK=1 && pnpm detox test",
+    "e2e:test": "pnpm --filter ledger-live-mobile-e2e-tests pnpm e2e:test",
```

Comment:
I guess these changes need to be properly communicated to devs also.
We also would need to change the documentation, and create a specific one for Speculos tests